css.js

```diff
@@ -1,36 +1,123 @@
 var loader = require("@loader");
 
+var isNode = typeof process === "object" && {}.toString.call(process) ===
+	"[object process]";
+var importRegEx = /@import [^uU]['"]?([^'"\)]*)['"]?/g;
+var resourceRegEx =  /url\(['"]?([^'"\)]*)['"]?\)/g;
+	
+var waitSeconds = (loader.cssOptions && loader.cssOptions.timeout)
+	? parseInt(loader.cssOptions.timeout, 10) : 60;
+var noop = function () {};
+var polling;
```

Comment:
This variable is going to be global for all modules, so if the user loads multiple CSS we have a potential race condition. We need to make these pollings be scoped inside of the fetch somehow.